onnxruntime/core/session/inference_session_utils.h

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/session/inference_session.h"
+#include "core/framework/session_options.h"
+#include "core/common/common.h"
+#include "single_include/nlohmann/json.hpp"
+
+using json = nlohmann::json;
+
+namespace onnxruntime {
+
+namespace inference_session_utils {
+
+static const std::string ort_config_key = "ort_config";
```

Comment:
Names of these constants seem off. see https://google.github.io/styleguide/cppguide.html#Constant_Names.

Reply:
Thanks, fixed.